SignalMessaging/profiles/OWSProfileManager.m

```diff
@@ -32,69 +33,6 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-// UserProfile properties may be read from any thread, but should
-// only be mutated when synchronized on the profile manager.
-@interface UserProfile : TSYapDatabaseObject
-
-@property (atomic, readonly) NSString *recipientId;
-@property (atomic, nullable) OWSAES256Key *profileKey;
-@property (atomic, nullable) NSString *profileName;
-@property (atomic, nullable) NSString *avatarUrlPath;
-// This filename is relative to OWSProfileManager.profileAvatarsDirPath.
-@property (atomic, nullable) NSString *avatarFileName;
-
-// This should reflect when either:
-//
-// * The last successful update finished.
-// * The current in-flight update began.
-@property (atomic, nullable) NSDate *lastUpdateDate;
-
-- (instancetype)init NS_UNAVAILABLE;
-
-@end
-
-#pragma mark -
-
-@implementation UserProfile
```

Comment:
Renamed `UserProfile` to `OWSUserProfile` and made it "all atomic + update with..."